### ISSUE TITLE
Feature suggestion: editing a fighter's victory theme

### ISSUE THREAD

**Issue by Segtendo** [Deinonychus71/Sma5hMusic#86]
A feature I'd like to see implemented some day is the ability to edit a fighter's victory theme.

For those reading and are unaware, the game tells what song to use for a fighter's victory theme, which is stored in `ui_bgm_db`, underneath the `fighter_jingle` list of the PRC file.

For example, for Mario's, it lists `z01_f_mario` for the data_name string, which refers to `bgm_z01_f_mario.nus3audio` in the game's music folder.

tl;dr I'd love to see a feature implemented where we can edit a fighter's victory theme, along with change the ones that share with other's like Fox/Wolf and Ryu/Ken and use new themes for them instead of sharing.

**Comment by Deinonychus71** [comment]
I do want to find the time to implement this at some point. I've been busy recently but I'll keep it in mind.
